src/main/java/de/codecrafter47/taboverlay/config/dsl/CustomPlaceholderConfiguration.java

```diff
@@ -317,7 +321,7 @@ public ColorAnimation() {
 
             OptionalInt finalDistance = distance;
             float finalSpeed = speed;
-            return builder.acquireData(() -> new CustomPlaceholderColorAnimation(textTemplate, colors, finalDistance, finalSpeed), TypeToken.STRING, textTemplate.requiresViewerContext());
+            return builder.acquireData(() -> new CustomPlaceholderColorAnimation(textTemplate, colors, formats.getValue(), finalDistance, finalSpeed), TypeToken.STRING, textTemplate.requiresViewerContext());
```

Comment:
This could throw a NullPointerException. I'd use something like `formats != null ? formats : ""`, then you can get rid of the null check in the `CustomPlaceholderColorAnimation` class.

Reply:
Changed by using the sanitize method for this.